sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3102,28 +3103,17 @@ def _exec_code_on_head(
         """Executes generated code on the head node."""
         style = colorama.Style
         fore = colorama.Fore
-        ssh_credentials = backend_utils.ssh_credential_from_yaml(
```

Comment:
Just curious, how much is removing this speeding things up?

Reply:
For a cloud (aws), it does not change the time for submitting the job in `_exec_code_on_head` from 2.65s to 2.64s (average over 3 times, not much difference).

For kubernetes, it reduces time from 9.92s to 6.87s (average over 3 times, 30% faster).